src/components/body/Body.jsx

```diff
@@ -1,8 +1,22 @@
 import PropTypes from 'prop-types'
+import styles from './Body.module.scss'
```

Comment:
Order the imports following this convention 

standard libraries
third party libraries
your own code imports
imports specific to the module (e.g. css, png, etc.)

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/order.md